bin/oref0-setup.sh

```diff
@@ -248,13 +248,16 @@ mkdir -p $HOME/src/
 if [ -d "$HOME/src/oref0/" ]; then
     echo "$HOME/src/oref0/ already exists; pulling latest"
     (cd ~/src/oref0 && git fetch && git pull) || die "Couldn't pull latest oref0"
+    echo (Re)installing oref0 installation from ~/src/oref0
+    cd $HOME/src/oref0/ && npm run global-install
```

Comment:
Do we need to re-install every time?  Or is there some quicker way we can check if we're up to date?  Maybe only do global-install if `git pull` updated anything?